modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -216,12 +217,37 @@ export default class MVTLayer extends TileLayer {
     return renderedFeatures;
   }
 
+  getVisibleTiles(tileCoords = 'local') {
```

Comment:
Why is `tileCoords` needed?

Reply:
Let's say `viewport.resolution` is = `'local'`, but user needs the tile data in WGS84 right?

Reply:
If you want the local coords, you access `tile.data`. If you want WGS84, you access `tile.dataInWorldCoordinates`. Both should always be available in any tile instance.